crates/bitwarden-core/src/key_management/mod.rs

```diff
@@ -18,6 +18,9 @@ mod crypto_client;
 #[cfg(feature = "internal")]
 pub use crypto_client::CryptoClient;
 
+mod security_state;
+pub use security_state::{SecurityState, SignedSecurityState};
```

Comment:
These can probably be internal-only
```suggestion
#[cfg(feature = "internal")]
mod security_state;
#[cfg(feature = "internal")]
pub use security_state::{SecurityState, SignedSecurityState};
```